packages/block-library/src/embed/util.js

```diff
@@ -98,7 +102,8 @@ export const createUpgradedEmbedBlock = (
 	// WordPress blocks can work on multiple sites, and so don't have patterns,
 	// so if we're in a WordPress block, assume the user has chosen it for a WordPress URL.
 	const isCurrentBlockWP =
-		providerNameSlug === WP_VARIATION.attributes.providerNameSlug ||
+		( WP_VARIATION &&
```

Comment:
If `WP_VARIATION` doesn't exist but `type=== WP_EMBED_TYPE` would result in true even if the wanted check would be to mark it `falsy` as we have unregistered the `WP` variation. This is the case you want to handle, no?

```
WP_VARIATION &&
( providerNameSlug === WP_VARIATION.attributes.providerNameSlug ||
	type === WP_EMBED_TYPE );
```

Reply:
Ah yeah, good point, thank you!